app/javascript/packages/document-capture/components/selfie-capture.scss

```diff
@@ -157,6 +157,12 @@ div.selfie-capture__label[tabindex='-1']:focus {
 .selfie-capture__capture-icon {
   height: 2rem;
   width: 2rem;
+
+  .usa-button:not(.usa-button--unstyled) > &.usa-icon:first-child {
+    // Unset icon placement where assumed to be adjacent to text, since the selfie capture button
+    // uses a standalone icon.
+    margin: 0;
```

Comment:
For reference:

https://github.com/18F/identity-idp/blob/883e767227a63b43a9a48450105d407054eaf48a/app/assets/stylesheets/components/_icon.scss#L3-L19